### DISCUSSION TITLE
Help Shape the Future of OpenFGA: Complete the OpenFGA Community Survey 2024!

### DISCUSSION THREAD

**Discussion by hello-caleb** [openfga/community#336] [Announcements]
Hi friends! We're running a [survey](https://www.surveymonkey.com/r/OPENFGA2024) to better understand how you all are using OpenFGA and what you'd like to see us shipping in the future. If you have a few minutes we'd love your input!

**Comment by dongniwang** [comment]
Closing this one since it's outdated. Thank you!
